troubleshooting/application-title-in-email-templates-wrong.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Application.Title in Email Template is Wrong
+description: You renamed the name of your project in Telerik Platform portal, but in emails Application.Title still the old name appears.
+type: troubleshooting
+page_title: How to Change the Application.Title in Email Templates
+slug: application-title-in-email-templates-wrong
+position:
+tags: backendservices, portal, email-notification
+teampulseid:
+ticketid: 1111828, 1072170, 1060801, 1054819, 1087785, 1100004, 1058035, 1087785
+pitsid:
+
+---
+## Environment
+<table>
+  <tr>
+    <td>Service</td>
+    <td>{{site.tp}}, {{site.bs}} {{site.bs-n}}</td>
+  </tr>
+</table>
+
+## Description
+After changing the application name in Telerik Platform portal, the old name {{Application.Title}} still appearing in all automated emails.
+
+## Cause
+Internal limitation. The {{Application.Title}} is not based on the application name field, but instead takes/gets its value from a different field (backend project name). The value in this field remains unchanged when you modify the application name from the portal UI.

```

Comment:
We'd rather simplify this. Limitation sounds like it may be removed. Suggestion:

The application title takes its name after the initial name of the app in {{site.tp}}. The value in this field remains unchanged when you modify the app name from the {{site.tp}} UI.